### ISSUE TITLE
Training details

### ISSUE THREAD

**Issue by Yzichen** [dongkwonjin/RVLD#15]
Thanks for your perfect work, I would like to know your training configuration for CLRNet, GANet on openlane, mainly learning rate, batch_size and epochs.
